code/modules/antagonists/revenant/revenant_abilities.dm

```diff
@@ -404,5 +404,59 @@
 		tray.set_weedlevel(rand(8, 10))
 		tray.set_toxic(rand(45, 55))
 
+/datum/action/cooldown/spell/aoe/revenant/haunt_object
+	name = "Haunt Object"
+	desc = "Empower nearby objects to you with ghostly energy, causing them to attack nearby mortals. \
+		Items closer to you are more likely to be haunted."
+	icon_icon = 'icons/mob/actions/actions_revenant.dmi'
+	button_icon_state = "r_haunt"
+	max_targets = 7
+	aoe_radius = 5
+
+	unlock_amount = 50
+	cast_amount = 40
+	stun_duration = 4 SECONDS
+	reveal_duration = 6 SECONDS
+
+	/// Static list of types that, if our haunted items are smacked with one of these, they will stop being haunted.
+	var/static/list/types_which_dispell_us = list(/obj/item/nullrod, /obj/item/storage/book/bible)
```

Comment:
This list actually isn't used anywhere, and the only reason it works in-game at the moment is because you have a backup list that has the same stuff as this list does.